cumulus_etl/chart_review/cli.py

```diff
@@ -94,6 +115,49 @@ def philter_notes(notes: Collection[LabelStudioNote], args: argparse.Namespace)
         note.text = philter.scrub_text(note.text)
 
 
+def group_notes_by_encounter(notes: Collection[LabelStudioNote]) -> list[LabelStudioNote]:
+    """
+    Gather all notes with the same encounter ID together into one note.
+
+    We also add little headers to each note with the Real & Anonymized docref IDs, to help reviewers.
+    We've found that reviewers appreciate both practices.
+    """
+    big_notes = []
```

Comment:
suggestion, feel free to disregard (especially because, i, personally, find **BIG NOTES** hilarious for reasons): concatenated_notes might be more descriptive? this trickles down through the rest of this function.

Reply:
:smile: I changed the `big_` prefix in this method to `grouped_` which felt easier to say and matches the function name too (`group_notes_by_encounter`).